posts/2021-06-11-kubernetes-operator-with-java-and-quarkus.adoc

```diff
@@ -0,0 +1,712 @@
+= Kubernetes Operator with Java and Quarkus
+
+== The goal of this blog post
+An Operator can be used for various reasons. The most common purposes of operators are managing database clusters, preparing database schemas, and restoring backups.
+Even though Kubernetes provides extensive support for multiple container management operations, it doesn't cover all potential cases related to application management. Some applications need the manual intervention of a human operator. A Kubernetes operator is an application that runs in the cluster and manages resources.
+It allows the transfer of operational knowledge of the human operator into software components. Operators can be used for multiple purposes such as restoring data, creating cloud resources, resilience testing, and many others.
+
+The purpose of this blog post is to demonstrate how to quickly write and deploy a simple Kubernetes operator using Java, and provide an example of tasks that can be performed using an operator.
+
+==== The benefits of choosing Java and Quarkus
```

Comment:
Maybe you can change this from a title into an intro to the different bullet points:

```suggestion
I decided to use a Quarkus service written in Java for the following reasons:
```